src/org/etools/j1939_84/controllers/part01/Part01Step07Controller.java

```diff
@@ -220,10 +220,10 @@ protected void run() throws Throwable {
 
         // 6.1.7.4.a. Destination Specific (DS) DM19 to each OBD ECU (plus all
         // ECUs that responded to global DM19).
```

Comment:
This collection of addresses needs to also include the addresses of the modules that responded to the global request.